api_spec/edge_api.json

```diff
@@ -91,6 +94,40 @@
                     }
                 }
             }
+        },
+        "/bulk-identities": {
+            "post": {
+                "tags": [],
+                "summary": "Bulk insert identities",
+                "description": "Bulk uploading identities and traits",
+                "requestBody": {
+                    "content": {
+                        "application/json": {
+                            "schema": {
+                                "type": "object",
+                                "example": {
+                                    "data": [
+                                        {
+                                            "$ref": "#/components/schemas/Sample"
+                                        },
+                                        {
+                                            "$ref": "#/components/schemas/Sample"
```

Comment:
It's not really realistic to provide the same data twice in this array. We should just remove this second instance. It's clear that it's an array from the rest of the information.